index.html

```diff
@@ -713,8 +718,12 @@ <h4 class="heading settled" data-level="3.1.3" id="attributes"><span class="secn
     <dt><dfn class="dfn-paneled idl-code" data-dfn-for="MediaStreamTrackProcessor" data-dfn-type="attribute" data-export id="dom-mediastreamtrackprocessor-readable"><code>readable</code></dfn>, <span> of type <a data-link-type="idl-name" href="https://streams.spec.whatwg.org/#readablestream" id="ref-for-readablestream①">ReadableStream</a></span>
     <dd>Allows reading the frames flowing through the <code class="idl"><a data-link-type="idl" href="https://www.w3.org/TR/mediacapture-streams/#dom-mediastreamtrack" id="ref-for-dom-mediastreamtrack⑤">MediaStreamTrack</a></code> stored
 in the <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-track" id="ref-for-dom-mediastreamtrackprocessor-track①">[track]</a></code> internal slot. If <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-track" id="ref-for-dom-mediastreamtrackprocessor-track②">[track]</a></code> is a video track, chunks read from <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-readable" id="ref-for-dom-mediastreamtrackprocessor-readable①">readable</a></code> will be <code class="idl"><a data-link-type="idl">VideoFrame</a></code> objects. If <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-track" id="ref-for-dom-mediastreamtrackprocessor-track③">[track]</a></code> is an audio track, chunks read from <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-readable" id="ref-for-dom-mediastreamtrackprocessor-readable②">readable</a></code> will produce <code class="idl"><a data-link-type="idl">AudioFrame</a></code> objects.
+If media frames are not read from <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-readable" id="ref-for-dom-mediastreamtrackprocessor-readable③">readable</a></code> quickly enough,
+the <code class="idl"><a data-link-type="idl">MedeiaStreamTrackProcessor</a></code> will internally buffer up to <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-maxbuffersize" id="ref-for-dom-mediastreamtrackprocessor-maxbuffersize①">[maxBufferSize]</a></code> of the frames produced by <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-track" id="ref-for-dom-mediastreamtrackprocessor-track④">[track]</a></code>. If the internal buffer
+is full, each time <code class="idl"><a data-link-type="idl" href="#dom-mediastreamtrackprocessor-track" id="ref-for-dom-mediastreamtrackprocessor-track⑤">[track]</a></code> produces a new frame, the oldest frame
+in the buffer MUST be dropped and the new frame MUST be added to the buffer. 
```

Comment:
Suggest adding a <p class=note> about detecting the fact that this has happened by noticing that there's a gap in the timestamps of the frames.

Reply:
Done